akvo/rsr/spa/app/components/AuditTrail.jsx

```diff
@@ -50,6 +50,12 @@ const DetailsValue = ({ data, textReport, disaggregations, scores }) => {
           </Col>
         </>
       ) : null}
+      {(data?.status === 'R' && data?.reviewNote) && (
+        <>
+          <Col><Text type="secondary">Comment</Text></Col>
+          <Col>{data.reviewNote}</Col>
```

Comment:
I think the `Col` elements are missing the `span` attribute

Reply:
ah Okay, let me try to fix it